### DISCUSSION TITLE
Check for updated version not working

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2152] [Show and tell]
This is currently not working for Beta and Stable channels.
It works for Alpha and Nightly channels.  

If you are using either of Beta or Stable channels, please update manually.
To do that, please download the following and install over your current version of xDrip:
https://github.com/NightscoutFoundation/xDrip/releases/download/2022.01.31/xDrip-plus-20220131-33be05a.apk

You can see the details of how to update normally here:  https://navid200.github.io/xDrip/docs/Updates.html

**Comment by petervanrijt** [comment]
Perhaps this is a workaround. I have written a public Telegram bot which checks for new versions of xDrip + AndroidAPS + Nightscout, and alerts when there are. This because my girlfriend is T1D and using the software, but I'm doing the tech/maintenance and not having the updates pushed on my screen. Checks for updates every hour. Created for personal use but quickly decided to publish it. All the best, Peter

[https://t.me/androidapsreleases](https://t.me/androidapsreleases)

**Reply by Navid200** [reply]
Thanks for sharing.

Just to be clear, there was an overflow issue and a decision had to be made how to address it.  A solution is already in place.  We are in a transition period now.  

Not long from now, when the next Beta and Stable releases are promoted, everything will work fine again.  Meaning, you will be able to check for update from any release, old or new, to find out if there is a new release available in either one of the update channels.

**Reply by petervanrijt** [reply]
Much respect for all the work you and other put in it to help us all. Thanks!!
